### DISCUSSION TITLE
groops auto quit without reporting error

### DISCUSSION THREAD

**Discussion by zdwangjian5** [groops-devs/groops#274] [Q&A]
After the first program ”InstrumentResampleMiscValues“ finishes, it does not loop and move on to the next InstrumentResampleMiscValues program , but exits after execution without reporting any errors. Why? Looking for your help! Thanks for
![image](https://github.com/user-attachments/assets/ea484df3-ddc9-4440-8088-9b8d4e5c82ac)

**Comment by zdwangjian5** [comment]
![Uploading image.png…]()
![image](https://github.com/user-attachments/assets/0888e80a-fee1-473a-8d96-d1811c6942f8)

**Reply by zdwangjian5** [reply]
Thank you, i have found the problem.

**Comment by tmayerguerr** [comment]
InstrumentResampleMiscValues is not a program developed by us. I cannot help you finding the bug.
regards
Torsten

**Reply by zdwangjian5** [reply]
Thank you, i have found the problem.
